### PR TITLE
5 update user documentation to match v2.1.1

### PR DESCRIPTION
**Pull request by jkbquabeck**
_What issues does this PR close?_
#5 

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [x] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: docs/product/user-documentation/Administration and Support.md [hunk 1]
```diff
@@ -53,9 +67,10 @@ As an Authority Admin you can additionally:
...
+// add image with all connectors overview
+![]()
 
 ## Ticket System
 
-In case of any need for support, e.g. when encountering malfunctions, problems finding information or questions regarding the provided information, click on “Support” to access the Fault Management system.
-Find further documentation [here](https://mobility-dataspace.online/).
\ No newline at end of file
+In case of any need for support, e.g. when encountering malfunctions, problems finding information or questions regarding the provided information, click on “MDS Support” to access the Fault Management system.
+Find further documentation [here](https://mobility-dataspace.online/).
```
Comment (jridderbusch) [inline docs/product/user-documentation/Administration and Support.md:72]:
```suggestion
Find further support [here](https://mobility-dataspace.online/).
```

File: docs/product/user-documentation/Manage Data Space components.md [hunk 2]
```diff
@@ -8,34 +8,34 @@ Besides the Data Space Authority roles, there are two other application roles th
...
 
 #### 3. Assign Service Partner Admin role to users within own organization
 
-Go to the "Profile" page.
-Select the list of users.
+Click on "My Organization" in the side bar menu.
+Head to the "Users & Roles" page.
 Select a user's profile of the user that shall become a Service Partner Admin.
 Edit the application role and change it to Service Partner Admin.
 Result: An additional Service Partner Admin has been created.
```
Comment (jridderbusch) [inline docs/product/user-documentation/Manage Data Space components.md]:
```suggestion
1. Click on "My Organization" in the side bar menu.
2. Head to the "Users & Roles" page.
3. Select a user's profile of the user that shall become a Service Partner Admin.
4. Edit the application role and change it to Service Partner Admin.
5. Result: An additional Service Partner Admin has been created.
```

File: docs/product/user-documentation/Manage Data Space components.md [hunk 3]
```diff
@@ -58,21 +58,22 @@ Currently the Operator Admin role grants the following rights in addition to a u
...
 
 #### 3. Assign Operator admin role to users within own organization
 
-Go to the "Profile" page.
-Select the list of users.
+Click on "My Organization" in the side bar menu.
+Head to the "Users & Roles" page.
 Select a user's profile of the user that shall become an Operator Admin.
 Edit the application role and change it to Operator Admin.
 Result: An additional Operator Admin has been created.
```
Comment (jridderbusch) [inline docs/product/user-documentation/Manage Data Space components.md:81]:
```suggestion
1. Click on "My Organization" in the side bar menu.
2. Head to the "Users & Roles" page.
3. Select a user's profile of the user that shall become an Operator Admin.
4. Edit the application role and change it to Operator Admin.
5. Result: An additional Operator Admin has been created.
```

File: docs/product/user-documentation/Manage Data Space components.md (hunk 2 above)
Comment (jkbquabeck) [reply docs/product/user-documentation/Manage Data Space components.md]: Duplicate (?)

Comment (jridderbusch) [reply docs/product/user-documentation/Manage Data Space components.md]: No, same file, different lines

**Review by jridderbusch — APPROVED** [review]
LGTM

**Pull request merged**